api/v1alpha1/api.go

```diff
@@ -277,6 +277,16 @@ type AIGatewayRouteRuleBackendRef struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:default=1
 	Weight *int32 `json:"weight,omitempty"`
+	// Priority is the priority of the AIServiceBackend. This sets the priority on the underlying endpoints.
+	// See: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/priority
+	// Note: This will override the `faillback` property of the underlying Envoy Gateway Backend
+	//
+	// Default is 0.
+	//
+	// +optional
+	// +kubebuilder:validation:Minimum=0
```

Comment:
it seems trivial but let's add the CEL validation test as @aabchoo commented above - you can take a look at tests/crdcel

Reply:
> it seems trivial but let's add the CEL validation test as @aabchoo commented above - you can take a look at tests/crdcel

I tried adding a CEL validation where `-1` was passed but this resulted in a parsing error as it cannot be parsed as an unsigned int. Since any valid unsigned int is a valid priority I am not sure we need a validation test. WDYT